texttospeech/list_voices/list_voices.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+/*
+Example usage:
+	go get -u cloud.google.com/go/texttospeech/apiv1
+	go run list_voices.go
+*/
+
+import (
+	"fmt"
+	"log"
+
+	"golang.org/x/net/context"
+
+	texttospeech "cloud.google.com/go/texttospeech/apiv1"
+	texttospeechpb "google.golang.org/genproto/googleapis/cloud/texttospeech/v1"
+)
+
+// [START tts_list_voices]
+func ListVoices() error {
+	ctx := context.Background()
+
+	client, err := texttospeech.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Performs the list voices request
+	request := texttospeechpb.ListVoicesRequest{}
+	response, err := client.ListVoices(ctx, &request)
+	if err != nil {
+		return err
+	}
+
+	for _, voice := range response.Voices {
+		// Display the voice's name. Example: tpc-vocoded
+		fmt.Printf("Name: %v\n", voice.Name)
+
+		// Display the supported language codes for this voice. Example: "en-US"
+		for languageCode := range voice.LanguageCodes {
+			fmt.Printf("  Supported language: %v\n", languageCode)
+		}
+
+		// SSML Voice Gender values from google.cloud.texttospeech.enums
+		ssmlVoiceGenders := []string{
```

Comment:
These actually have enums in the protobuf package.

https://godoc.org/google.golang.org/genproto/googleapis/cloud/texttospeech/v1#SsmlVoiceGender

The regular `String()` method should print the pretty name.

Reply:
Thanks, I didn't knew a `String()` method existed for enums